src/main/kotlin/controller/RacingCarGame.kt

```diff
@@ -0,0 +1,27 @@
+package controller
+
+import model.RacingCar
+import view.InputView
+import view.ResultView
+
+fun main() {
+    RacingCarGame(InputView(), ResultView()).run()
+}
+
+class RacingCarGame(private val inputView: InputView, private val resultView: ResultView) {
+    fun run() {
+        val cars = inputView.enterAmountOfCars()
+        val rounds = inputView.enterAmountOfRounds()
+
+        val racingCars: List<RacingCar> = List(cars) { RacingCar() }
```

Comment:
Have you thought about using a "First-Class Collection"?
It could offer some benefits. What do you think it might improve in your code?